app/views/layouts/blacklight/base.html.erb

```diff
@@ -31,17 +31,17 @@
       <%= link_to t('blacklight.skip_links.main_content'), '#main-container', class: 'element-invisible element-focusable rounded-bottom py-2 px-3', data: { turbolinks: 'false' } %>
       <%= content_for(:skip_links) %>
     </nav>
-    <%= render partial: 'shared/header_navbar' %>
+    <%= render blacklight_config.header_component.new(blacklight_config: blacklight_config) %>
```

Comment:
This is a pretty big extension point for a lot of applications; I wonder if it'd be nicer to downstream apps if we keep rendering `shared/header_navbar` and render the header component inside it?